image_view/scripts/extract_images_sync

```diff
@@ -51,10 +51,19 @@ class ExtractImagesSync(Node):
         super().__init__('extract_images_sync')
         self.get_logger().info('Extract_Images_Sync Node has been started')
         self.seq = 0
+        self.time = self.get_clock().now()
+
         self.fname_fmt = self.declare_parameter(
-            'filename_format', 'frame%04i_%i.jpg').value   
+            'filename_format', 'frame%04i_%i.jpg').value
+
+        # Do not scale dynamically by default.
         self.do_dynamic_scaling = self.declare_parameter(
             'do_dynamic_scaling', False).value
+
+        # Limit the throughput to 10fps by default.
+        self.sec_per_frame = self.declare_parameter(
+            'sec_per_frame', 0.1).value
```

Comment:
we should probably not change default behavior - so this should probably be set to 0 by default